src/components/shoppingList/shoppingList.module.css

```diff
@@ -14,12 +14,33 @@
 
 .trigger {
   cursor: pointer;
+  display: grid;
+  grid-template-columns: max-content 1fr;
+}
+
+.icons {
+  margin: 8px 0;
+  display: grid;
+}
+
+.icon {
+  color: var(--text-color-primary);
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  margin-left: 4px;
+  margin-right: 4px;
+}
+
+.icon:hover {
+  color: var(--scheme-color-lightest);
```

Comment:
I don't like this on schemes where the primary text colour is darker than the main scheme colour. In these cases, the icon abruptly goes from a dark colour to a much lighter colour in a way that doesn't look intentional. This is annoying but it is the way it worked in the previous implementation and I couldn't think of a super easy way to make it better, so I'm leaving it for now and will make a card to improve it.

The obvious option would've been to set the `cursor` to `pointer`, however, it is set to `pointer` for the entire `.trigger` element, so there would be no change when the user moved the mouse over the icon.